include/zephyr/posix/net/if.h

```diff
@@ -8,4 +8,12 @@
 
 #include <zephyr/net/socket.h>
 
+#ifdef __cplusplus
```

Comment:
Does this do anything?

Reply:
It's just there so that it isn't forgotten in the future.